### ISSUE TITLE
[Question] Where put custom aliases and mixins?

### ISSUE THREAD

**Issue by nestquik** [AmmyUI/AmmyUI#115]

**Comment by ionoy** [comment]
Good stuff 👍

**Comment by nestquik** [comment]
@ionoy I'm also curious. How can this ([Diploma](https://github.com/nestquik/diploma/blob/cbad720fbadde50b79c080e077ca8b0020705376/src/Diploma/Views/RegisterView.ammy#L82)) could be translated in Ammy?
`l:Static` is kinda `System.Windows.Markup.StaticExtension` the implementation of it is here [Gu.Localization](
https://github.com/JohanLarsson/Gu.Localization/blob/2898a2c4df12c5027cefb38c6cf085b5d68f3398/Gu.Wpf.Localization/MarkupExtensions/StaticExtension.cs#L18).

**Comment by Tyrrrz** [comment]
Yeah i was also wondering about StaticExtensions, more specifically [this](https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/wiki/Transitions#transitioningcontent) (the `OpeningEffectsOffset` thing). Is it possible to achieve with AmmyUI currently without having to write XAML syntax?

**Comment by ionoy** [comment]
Just import `materialDesign` with typeless property:

```
"xmlns:materialDesign": "clr-namespace: ..."

OpeningEffectsOffset: "{materialDesign:IndexedItemOffsetMultiplier 0:0:0.05}"
```

or you can do something like this:

```
<TransitioningContent.OpeningEffectsOffset>
  0:0:0.05
</TransitioningContent.OpeningEffectsOffset>
```
